homework/style.css

```diff
@@ -1,3 +1,90 @@
 .alert-error {
     color: red;
-}
\ No newline at end of file
+    ba
+}
+
+body {
+  background-color: rgba(142, 93, 93, 1);
+}
+#root {
+  display: flex;
+   flex-direction: column;
+  align-items: flex-start;
+    justify-content: center;
+}
+.header {
+  background-color: #444;
+  width: 100%;
+}
+.h1 {
+  text-align: center;
+
+}
+.container {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  margin: 1 rem;
+}
+.eachContrib {
+    display: flex;
+  flex-direction: row;
+}
+.left-div {
+  padding: 16px;
+  margin-right: 16px;
+  background-color: beige;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+
+.right-div {
+ flex: column;
+ background-color: beige;
+ display: flex;
+ flex-direction:column;
+
+}
+.select {
+    display: flex;
+    flex-direction: row;
+    font-size: 16px;
+    font-family: sans-serif;
+    font-weight: 700;
+    color: #444;
+    line-height: 1.3;
+    padding: .6em 1.4em .5em .8em;
+    width: 20%;
+    max-width: 100%; 
+    box-sizing: border-box;
+    margin: 0;
+    border: 1px solid #aaa;
+    box-shadow: 0 1px 0 1px rgba(0,0,0,.04);
+    border-radius: .5em;
+    -moz-appearance: none;
+    -webkit-appearance: none;
+    appearance: none;
```

Comment:
Nice! Didn't know that `appearance: none;`  was a thing, so I learned something new today :)